terraform.tfvars.j2

```diff
@@ -1,2 +1,3 @@
+assume_role           = "administrator"
```

Comment:
I think this is causing the PR to fail in concourse

Reply:
It can go.  It's required for local apply, and is present in most TF code bases.